internal/service/streamconnection/resource_stream_connection_migration_test.go

```diff
@@ -81,7 +81,7 @@ func TestAccMigrationStreamRSStreamConnection_cluster(t *testing.T) {
 		clusterInfo  = acc.GetClusterInfo(nil)
 		instanceName = acc.RandomName()
 	)
-	mig.SkipIfVersionBelow(t, "1.14.0")
+	mig.SkipIfVersionBelow(t, "1.15.2")
```

Comment:
why do we need to change this?

Reply:
Because DbRoleToExecute is a new attribute that wasn't available in previous versions of our provider. So this test should not be executed for let's say 1.15.0